apiary.apib

```diff
@@ -4177,7 +4177,7 @@ Time tracking allows users to record time spent on tasks or projects and the typ
 
 ### timeTracking.list [GET /timeTracking.list]
 
-Get a list of tracked time.
+Get a list of tracked time. Per day, time tracking entries with an automatically generated start time are displayed first (duration time tracking), followed by entries with a user generated start and stop time (start-stop time tracking), sorted ascending by start time.
```

Comment:
> with an automatically generated start time

Is this correct though? Duration time tracking has no start time, right?

Reply:
It does. The api always returns a start time and a stop time.